app/core/store/modules/seasonalLeague.js

```diff
@@ -0,0 +1,215 @@
+import { getLeaderboard, getMyRank } from '../../api/leaderboard'
+import { fetchMySessions } from '../../api/level-sessions'
+
+// Level called: Void-Rush
+// const currentSeasonalLevelOriginal = '5fad3d71bb7075d1dd20a1c0'
+
+// Zero-sum for testing
+const currentSeasonalLevelOriginal = '550363b4ec31df9c691ab629'
+
+/**
+ * We want to be able to fetch and store rankings for
+ * various levels. I.e.
+ * https://codecombat.com/db/level/5fad3d71bb7075d1dd20a1c0/rankings?order=-1&scoreOffset=1000000&limit=20&team=humans&_=1607469435140
+ */
+export default {
+  namespaced: true,
+  state: {
+    loading: false,
+    mySession: [],
```

Comment:
Why is this initialized to an array initially? It's a singular session (unlike on previous ladders, where there might be humans and ogres versions).

Reply:
great catch!